tests/test_helpers.py

```diff
@@ -17,6 +17,11 @@
       "--date", "2020-04-29", "-c", "afriend", "-d", "more work",
       "-m", "ascopatz", "-g", "SymPy-1.1", "-u", "2021-01-01"],
      "20ly_newprojectum has been added in projecta\n"),
+    (["helper", "a_expense", "159.18", "Payment to google", "-b", "-g", "dmref15", "mrsec14"],
+     "Payment to google has been added to expenses\n"),
+    (["helper", "a_expense", "159.18", "Payment to google", "-b", "-y",
```

Comment:
I don't think you can currently run multiple tests with the same output name

Reply:
Thanks. I will update my tests.